test/playwright.spec.js

```diff
@@ -103,6 +104,8 @@ module.exports.describe = ({testRunner, product, playwrightPath}) => {
     beforeAll(async state => {
       state.browser = await browserType.launch(defaultBrowserOptions);
       state.browserServer = state.browser.__server__;
+      state._stdout = readline.createInterface({ input: state.browserServer.process().stdout });
```

Comment:
does this fail with DUMPIO=true?

Reply:
It would never fail. What you are hinting at is 'ignore' stdio spawn flag that we used to have. it is long gone.